### PR TITLE
[BACKLOG-37696] Moving the Rest related custome analyzer to new Rest …

### PR DESCRIPTION
**Pull request by pdesai16**
…plugin in PDI.

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **818**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/metaverse-plugin/job/PR-678/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by pdesai16** [comment]
This is related to PR https://github.com/pentaho/pentaho-kettle/pull/8916 that moves the Rest classes to new Plugin.
Both of these PR are waiting for fixes to bugs [PDI-19777](https://hv-eng.atlassian.net/browse/PDI-19777) and [PDI-14825](https://hv-eng.atlassian.net/browse/PDI-14825) to get checked in first

File: core/src/it/java/org/pentaho/metaverse/MetaverseValidationIT.java [hunk 1]
```diff
@@ -25,8 +25,7 @@
 import org.apache.commons.collections.IteratorUtils;
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang.StringUtils;
-import org.junit.AfterClass;
-import org.junit.Test;
+import org.junit.*;
```
Comment (niehusa) [inline core/src/it/java/org/pentaho/metaverse/MetaverseValidationIT.java:28]: Do we need to optimize import here?

**Review by niehusa — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 30s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **818**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/metaverse-plugin/job/PR-678/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **818**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/metaverse-plugin/job/PR-678/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by abryant-hv — APPROVED** [review]

**Review by rkimmel-hv — APPROVED** [review]

File: core/src/it/java/org/pentaho/metaverse/MetaverseValidationIT.java [hunk 2]
```diff
@@ -27,6 +27,7 @@
 import org.apache.commons.lang.StringUtils;
 import org.junit.AfterClass;
 import org.junit.Test;
+import org.junit.Ignore;
```
Comment (niehusa) [inline core/src/it/java/org/pentaho/metaverse/MetaverseValidationIT.java:30]: Could have removed this import since we're no longer ignoring any test.

**Review by niehusa — APPROVED** [review]

**Review by peterrinehart — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m 23s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **818**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/metaverse-plugin/job/PR-678/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**